api/src/movielist/movielist.controller.ts

```diff
@@ -0,0 +1,56 @@
+import {
+  Controller,
+  BadRequestException,
+  Body,
+  Post,
+  Req,
+  UseInterceptors,
+  UseGuards,
+  Param,
+  Get,
+  Patch,
+} from '@nestjs/common';
+import { RemoveListCreateFieldsInterceptor } from '../list/interceptor/remove-list-create-fields.interceptor';
+import { Request } from 'express';
+import { CreateListDto } from '../list/dto/create-list.dto';
+import { MovielistService } from './movielist.service';
+import { ListAuthGuard } from '../list/guard/list.guard';
+import { UpdateMovieListDto } from '../movie/dto/update-movie-list.dto';
+
+@Controller('movielist')
+export class MovielistController {
+  constructor(private movieListService: MovielistService) {}
+
+  @UseGuards(ListAuthGuard)
+  @Get('/:id/sharees')
```

Comment:
We have this same endpoint in the `List` module. I wonder if we should only keep the other one since this operation doesn't require movies at all.

Reply:
A few things for this:
- This should probably return a list of `ids` that'll reference `sharees`.
- This should happen in the `List` module.
- We can probably use the `Movie` moduie (`Movie` service with filters), to get movies watched by a user in a list.

Reply:
Added in latest commit, let me know!